### PR TITLE
Add ember sem-release config; tslint

### PR DESCRIPTION
**Pull request by damassi**
Updates package with `conventional-changelog-ember` which mirrors Reaction, as well as Reaction's `tslint.json` config.

### CODE REVIEW

File: src/elements/Typography.tsx [hunk 1]
```diff
@@ -37,7 +37,7 @@ const Text = styled.Text.attrs<TextProps>({})`
   ${maxWidth};
 `
 
-interface TypographyProps {
+export interface TypographyProps {
```
Comment (damassi) [inline src/elements/Typography.tsx]: @zephraph - this is needed for `tsc --emit`, which will automatically emit type definition files for consumers.

File: tslint.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    "member-access": [false],
+    "member-ordering": false,
+    "no-unused-expression": false,
+    "no-var-requires": false,
+    "object-literal-sort-keys": false,
+    "ordered-imports": false,
+    "no-namespace": [true, "allow-declarations"],
+    "trailing-comma": false,
+    "variable-name": [true, "ban-keywords", "allow-leading-underscore"]
+  }
```
Comment (damassi) [inline tslint.json]: Taken verbatim from reaction. We can can clean this up at some point.

Comment (zephraph) [reply tslint.json]: Yeah, there's a lot of config stuff shared across projects. Would be good to figure out a good way to share those.

Comment (damassi) [reply tslint.json]: Totally agree, this is also dup'd in force now too

**Pull request merged**